app/models/clickdata.py

```diff
@@ -35,7 +30,7 @@ class ClickdataModel(BaseModel):
 
     @staticmethod
     @xray_recorder.capture_async('models.clickdata.get')
-    async def get(module: RecommendationModules, item_list: List[str]) -> Dict[str, 'ClickdataModel']:
+    async def get(slate_id: str, item_list: List[str]) -> Dict[str, 'ClickdataModel']:
```

Comment:
It looks to me like `RecommendationModel` needs to know `slate_id` to use the current call path?  I'd rather make the changes now using slates rather than a stopgap.

Reply:
additionally I am working in dev on the clickdata lambda (that's in use now) to make it use slate_ids.  so I think this would give us a chance to kick the tires on this PR with the topic pages before the andorid experiment.  and when the new clickdata table G is building goes live we should just be switching the table name to move everything over.